### ISSUE TITLE
A11y_AzureDataStudio_Delete connection_Usability:Once connection is deleted the Deletion status message is not announced by screen reader

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio#23073]
### GitHub Tags:
#A11yusable; #A11yTCS; #A11ySev4; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #Win32;  #NVDA; #Screen Reader; #Benchmark; #JAWS;

### Environment and OS details:
Azure Data Studio Version: 1.42.0 (user setup)
Windows 11 Enterprise Version	22H2
Installed on	‎28-‎12-‎2022
OS build	22621.1413
Experience	Windows Feature Experience Pack 1000.22639.1000.0
NVDA Version 2023.1
JAWS Version 2023
### Pre-Requisite:

1. Install Azure data studio From here https://learn.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio?view=sql-server-ver16&tabs=redhat-install%2Credhat-uninstall
2. Once installation done , Install MySQL Extension in Azure data studio
3. Tab Till  new connection and press enter 
4. Enter a valid server name username and password and press connect
### Repro Steps:

1. Launch Azure Data Studio. and Turn on nVDA
2. Select the Connections icon (if not already selected) from left bar.
3. Right click on any of the servers from the Servers viewlet and select "Delete connection"
4. The selected connection should be deleted
5.Verify that once connection is deleted the status is announced by screen reader or not

### Actual Result:

Once connection is deleted the Deletion status message is not announced by screen reader
**Issue repro with JAWS**
### Expected Result:
NVDA should announce Deletion status message.
### Attachment:

![Video0 (11)](https://user-images.githubusercontent.com/100189407/229684907-4e4ef08c-befd-4d3c-927e-5b848adb0317.gif)
![Screenshot (83)](https://user-images.githubusercontent.com/100189407/229722897-ddb3c7c1-3807-4c73-a63a-8ae36078ce17.png)

**Comment by VijayaBharathyA** [comment]
Primary bug:https://github.com/microsoft/azuredatastudio-mysql/issues/193
